applications/system/js_app/types/global.d.ts

```diff
@@ -18,15 +18,53 @@ declare function print(...args: any[]): void;
  */
 declare function toString(value: number, base?: number): string;
 
+/**
+ * @brief Converts a string to a number
+ * @param text The string to convert to a number
+ */
+declare function parseInt(text: string): number;
+
+/**
+ * @brief Transforms a string to upper case
+ * @param text The string to transforms to upper case
+ */
+declare function toUpperCase(text: string): string;
```

Comment:
Like you said, these probably belong in the String class.

Reply:
Any thoughts on moving toString()? That's how it is in ecmascript too and would be good for consistency I think

Reply:
moved string things